### DISCUSSION TITLE
In the discussion forum the "View count" in question asked, increases every time we click on bookmark, Upvote and downvote.

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#641] [General Discussion]
We are using the 4.6 version of Sunbird 
We have created one question in the discussion forum and when we click on the bookmark, upvote, or downvote the view count increases. 
We have debugged a few things there
**This is the curl that is getting called every time we click on the bookmark, Upvote, and downvote.**

curl 'https://sunbirdsaas.com/discussion/topic/25/what-is-rejuvenation-of-water-bodies?page=1' \
  -H 'Accept: application/json' \
  -H 'Accept-Language: en-GB,en-US;q=0.9,en;q=0.8' \
  -H 'Access-Control-Allow-Origin: *' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json, text/plain' \
  -H 'Cookie: connect.sid=s%3AAMLc3pO5CTdBU0RqKqztoQfXnpVMLsNz.ehfzmS75S4E%2FBe6LaReDo1ZM1RO1LbzwoVeSyZ3Qj8M' \
  -H 'If-None-Match: W/"2b71-+Po2yKwpSv1xW28VZoEfqqOJois"' \
  -H 'Referer: https://sunbirdsaas.com/discussion-forum/topic/25/what-is-rejuvenation-of-water-bodies?categories=%7B%22result%22:%5B%2242%22%5D%7D&userId=26' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36' \
  -H 'X-App-Id: production.production.portal' \
  -H 'X-Channel-Id: 01373476834370355257' \
  -H 'X-Device-Id: 87195a8edbba640e39096f3c7108bff9' \
  -H 'sec-ch-ua: "Chromium";v="118", "Google Chrome";v="118", "Not=A?Brand";v="99"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Linux"' \
  --compressed

https://github.com/Sunbird-Ed/Community/assets/109504832/23eaec7b-c978-404d-9458-70a53d634285

And is the video and screenshot for refrence
![image (19)](https://github.com/Sunbird-Ed/Community/assets/109504832/1ad72252-a6e6-4640-b2cb-e4b43a62bc4c)
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by ashokreddy1208** [comment]
@sachinb80 Thanks for raising this query. This is the current capability. If you are interested to make these changes and contribute we can help you.

cc @Krishnaj20 @vinukumar-vs
